2.0.1 contains raw JSX

<img width="814" height="91" alt="Image" src="https://github.com/user-attachments/assets/7a410e41-6246-4e3c-818f-4582f2214342" />

https://www.npmjs.com/package/vitest-browser-react?activeTab=code

```
Error: Error during dependency optimization:

✘ [ERROR] The JSX syntax extension is not currently enabled

    .yarn/__virtual__/vitest-browser-react-virtual-6cc1e2ad98/2/.yarn/berry/cache/vitest-browser-react-npm-2.0.1-55ecd09a4f-10c0.zip/node_modules/vitest-browser-react/dist/pure-3-75rOIt.js:70:58:
      70 │ ...nder, unmount } = await render(<TestComponent renderCallbackPro...
         ╵                                   ^
```

presumably caused by https://github.com/vitest-community/vitest-browser-react/pull/30

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,9 +1,7 @@
 {
   "compilerOptions": {
     "target": "ESNext",
-    "jsx": "preserve",
-    "jsxFactory": "React.createElement",
-    "jsxImportSource": "react",
+    "jsx": "react-jsx",
     "lib": ["esnext", "dom", "DOM.Iterable"],
     "module": "ESNext",
     "moduleResolution": "Bundler",

